cli/flox-rust-sdk/src/providers/buildenv.rs

```diff
@@ -1150,26 +1195,30 @@ mod realise_nixpkgs_tests {
         assert!(result.is_ok(), "{}", result.unwrap_err());
     }
 
-    #[test]
-    fn nixpkgs_published_pkg_no_matching_response() {
-        let locked_package = locked_published_package(None);
-        let mut client = MockClient::new(None::<String>).unwrap();
-        let mut resp = StoreInfoResponse {
-            items: std::collections::HashMap::new(),
-        };
-
-        resp.items
-            .insert(locked_package.outputs["out"].clone(), vec![StoreInfo {
-                url: "https://example.com".to_string(),
-            }]);
-        client.push_store_info_response(resp);
-
-        let buildenv = buildenv_instance();
-        let subst_resp = buildenv
-            .try_substitute_published_pkg(&client, &locked_package)
-            .unwrap();
-        assert!(!subst_resp);
-    }
+    // FIXME: uncomment this once the catalog can tell us which stores need auth
```

Comment:
Reviewer note: see the comment on why this is commented out at the moment. Just making the list of `StoreInfo`s empty duplicates the test below it.